mycore-solr/src/main/resources/components/solr/config/mycore.properties

```diff
@@ -46,6 +46,7 @@ MCR.Solr.IndexHandler.Factory=org.mycore.solr.index.handlers.MCRSolrLazyInputDoc
 MCR.URIResolver.xslIncludes.solr-export=
 
 MCR.URIResolver.xslIncludes.components=%MCR.URIResolver.xslIncludes.components%,solr-layout-utils.xsl
+# we do not provide a xslt 3.0 for the solr-layout-utils.xsl in components-3
```

Comment:
I usually put hints above the property?

Reply:
i put it in the place where the property for the solr-layout-utils.xsl would be.